src/twisted/newsfragments/12071.doc

```diff
@@ -0,0 +1 @@
+Added documentation on when ``spawnProcess`` uses ``os.posix_spawnp``.
```

Comment:
This should be like `twisted.internet.interfaces.IReactorProcess now documents certain details of the POSIX-based implementation included with Twisted.`

Or something similar if the docs move elsewhere.

Reply:
Applied.